src/foundation/reboot.css

```diff
@@ -247,3 +247,11 @@ summary {
 [hidden] {
   display: none !important;
 }
+
+body,
+html {
+  display: block;
+  width: 100%;
+  padding: 0;
+  margin: 0;
```

Comment:
[Check it here](https://github.com/necolas/normalize.css/blob/master/normalize.css#L24)